scripts/database.sql

```diff
@@ -179,7 +180,9 @@ CREATE TABLE Lanefile (
 	-- this is some simple change tracking
 	-- on every change a new Lanefile is stored, with the old contents and original_id referencing the real Lanefile
 	original_id    int        NULL REFERENCES Lanefile (id),
-	changed_date   timestamp  NULL -- the date the change was made	
+	changed_date   timestamp  NULL, -- the date the change was made
+
+	required_roles text NULL DEFAULT NULL	-- Roles required to access the Lanefile
```

Comment:
LaneFiles can be shared between lanes, so how does this work when a LaneFile is used by multiple lanes?

Also from what I can see there's no propagation of required roles from lanes to lane files, which means you'll have to edit the required roles manually for each file you want someone to be able to access. Is this the intended behavior?

Access management between shared resources get complicated quick 👿